ticdc/manage-ticdc.md

```diff
@@ -305,7 +374,37 @@ cdc cli changefeed remove --pd=http://10.0.10.25:2379 --changefeed-id simple-rep
 
 In the above command:
 
-- `--changefeed=uuid` represents the ID of the `changefeed` that corresponds to the replication task you want to remove.
+- `--changefeed-id=uuid` represents the ID of the `changefeed` that corresponds to the replication task you want to remove.
+
+After the replication task is removed, the state information of the task will be retained for 24 hours, mainly used for recording the replication checkpoint. Within this 24 hours, you cannot create a replication task of the same name.
+
+If you want to completely remove the task information, you can specify the `--force` or `-f` argument in the command. Then all information of the `changefeed` will be removed, and you can immediately create a `changefeed` of the same name.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed remove --pd=http://10.0.10.25:2379 --changefeed-id simple-replication-task --force
+```
+
+### Update task configuration
+
+Starting from v4.0.4, TiCDC supports modifying the configuration of the replication task (not dynamically). To modify the `changefeed` configuration, pause the task, modify the configuration, and then resume the task.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+cdc cli changefeed pause -c test-cf
+cdc cli changefeed update -c test-cf --sink-uri="mysql://127.0.0.1:3306/?max-txn-row=20&worker-number=8" --config=changefeed.toml
+cdc cli changefeed resume -c test-cf
+```
+
+Currently, you can modify the following configuration items:
+
+- `sink-uri` of the `changefeed`.
+- The `changefeed` configuration file and all configuration items in the file.
+- Whether to use the file sorting feature and the sorting directory.
+- The time zone used in the `changefeed`.
```

Comment:
```suggestion
- The time zone used in the `changefeed`.
```
Please remove this line too, it is my fault. We also need to update the Chinese document